crates/pink/src/snapshots/pink__runtime__detect_parameter_changes.snap

```diff
@@ -87,7 +88,7 @@ expression: "(<PinkRuntime as frame_system::Config>::BlockWeights::get(),\n    <
             payload_len: 16384,
         },
         instruction_weights: InstructionWeights {
-            version: "3 ps",
+            version: "4 ps",
```

Comment:
@kvinwang Caution

Reply:
It's OK since we haven't published the first version.